contrib/cloud.google.com/go/pubsub.v1/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/DataDog/dd-trace-go/contrib/cloud.google.com/go/pubsub.v1
+
+go 1.12
+
+require (
+	cloud.google.com/go/pubsub v1.4.0
+	github.com/Microsoft/go-winio v0.5.0 // indirect
+	github.com/opentracing/opentracing-go v1.2.0 // indirect
+	github.com/philhofer/fwd v1.1.1 // indirect
+	github.com/stretchr/testify v1.7.0
+	google.golang.org/api v0.46.0
+	google.golang.org/grpc v1.37.0
+	gopkg.in/DataDog/dd-trace-go.v1 v1.32.0-alpha.13
```

Comment:
We should have some sort of mechanism that warns us when we release a new version and we forget to bump the integrations. Perhaps a non-blocking CI check (like codecov) would be nice.

Reply:
We may not need to keep the integrations' mod files up to date with the latest tracer version. I don't see harm in leaving the minimum compatible version.

I  do think we should have the submodule tests replace their copy of dd-trace-go with the "local" copy, so we know when a change in dd-trace-go breaks submodules before we merge.

Reply:
> We may not need to keep the integrations' mod files up to date with the latest tracer version. I don't see harm in leaving the minimum compatible version.

Are you sure? When we fix critical bugs, or add new product features (such as stats computation for example), forgetting to bump the version here will cause this module to not benefit of that feature, resulting in a broken product. To me it seems quite critical to be on the edge.

> I do think we should have the submodule tests replace their copy of dd-trace-go with the "local" copy, so we know when a change in dd-trace-go breaks submodules before we merge.

I think the submodule should always be tested against the version in `go.mod`, because that's what runs on the user's machine. At the same time, I do agree that any changes made to the trunk of dd-trace-go should have integration tests run against it to ensure all ground is covered.

As we have previously discovered, there is quite a bit of ground to cover here and things to consider.